### PR TITLE
Make alert colors match Figma examples

### PR DESCRIPTION
**Pull request by Baggerone**
Also reverted previous color change of the umbrella alert icon

### CODE REVIEW

**Comment by Baggerone** [comment]
Example (not the red and pink colors)

image

File: application/templates/mail/_alert.plush.html [hunk 1]
```diff
@@ -1,35 +1,67 @@
 
-<div style="margin:16px 0px; padding:8px; border-radius: 8px; background: #EBEEF2;">
+<%
+  let bgColor = "#EEEEEE"
+  let alertColor = "#333333"
+  let alertSrc = ""
+  let alertAlt = ""
+
+  if ( alert_icon == "beach_access" ) {
+  	let bgColor = "#EBFFF7"
```
Comment (jason-jackson) [inline application/templates/mail/_alert.plush.html:9]: - Do we need these `let` in the if statements?
- Would it be cleaner to use a map?
- Would it be better to send generate this outside this file and pass it into it?

Comment (Schparky) [reply application/templates/mail/_alert.plush.html:9]: something like this, Jason?
```
const alerts = {
  "beach_access": {
    {
      "bgColor": "#EBFFF7",
      "alertColor": "#028353",
...
    }
  },
}
...
<img src="cid:<%= alerts[alert_icon][alertSrc] %>" alt="<%= alerts[alert_icon][alertAlt] %>" style="width:100%;"/>
```

**Review by Schparky — APPROVED** [review]

**Comment by jason-jackson** [comment]
Yes, @Schparky, though maybe we should also consider something more descriptive for names. `beach_access` is rather ambiguous and could cause confusion as it describes the icon, not the actual alert.

File: application/templates/mail/_alert.plush.html [hunk 2]
```diff
@@ -1,35 +1,84 @@
 
-<div style="margin:16px 0px; padding:8px; border-radius: 8px; background: #EBEEF2;">
+<%
+	let alertStyles = {
+    	beach_access: {
+  			bgColor: "#EBFFF7",
+  			alertColor: "#028353",
+  			alertSrc: "beach_access",
```
Comment (jason-jackson) [inline application/templates/mail/_alert.plush.html:7]: If wanted, you could omit this, as it's the same as alert_icon variable.

**Review by jason-jackson — APPROVED** [review]

**Pull request merged**